api/bucketsd/service.go

```diff
@@ -916,16 +916,7 @@ func (s *Service) SetPath(ctx context.Context, req *pb.SetPathRequest) (res *pb.
 	})
 	buck.UnsetMetadataWithPrefix(destPath + "/")
 
-	txn, err := s.Buckets.WriteTxn(ctx, dbID, tdb.WithToken(dbToken))
```

Comment:
The txns were causing reads to be blocked due to the nature of the `RWMutex` at the `go-threads` level. Now that we have the bucket level semaphore-based locking, no need for these anyway.

Reply:
Yes excellent, I think this is going to help a bunch with some issues we've been seeing in practice.

Reply:
Well, the bucket level semaphore-based locking stuff that is.